src/config.c

```diff
@@ -180,12 +180,21 @@ void parse_config (GromitData *data)
   g_scanner_scope_add_symbol (scanner, 2, "color",     (gpointer) 2);
   g_scanner_scope_add_symbol (scanner, 2, "arrowsize", (gpointer) 3);
   g_scanner_scope_add_symbol (scanner, 2, "minsize",   (gpointer) 4);
+  g_scanner_scope_add_symbol (scanner, 2, "maxsize",   (gpointer) 5);
 
   g_scanner_set_scope (scanner, 0);
   scanner->config->scope_0_fallback = 0;
 
   g_scanner_input_file (scanner, file);
 
+  /* defaults */
```

Comment:
Why thouch this part of the code? is it related?

Reply:
Sorry, I'm a bit confused by the github UI. Do you mean the lines 126 and 183, or 190-197?

Reply:
If you mean 126 and 183, I assume this is what's needed to add the parameter to configuration file, correct? As for 190-197, my tests indicated that if it's inside of the loop on line 199, the maxsize parameter gets overwritten to the default value every time. I don't know if you have any specific reason to have the defaults inside the parsing loop, but if you do then I'll do my best to work around that.

Reply:
Well it should have a default value every time _unless_ one is coming from the config. I suspect that's related to my comment in callbacks.c